### DISCUSSION TITLE
Slow working sidecar

### DISCUSSION THREAD

**Discussion by Miloszzz2** [tauri-apps/tauri#9226] [Q&A] [answered]
Hi,
I have a problem with sidecar. I'm running sidecar (.exe generated from .py) and it take much more time than running command with python.exe (using std::process). I have also tested running .exe without tauri sidecar. Do you have any solutions for that? I can also mention that this binary creates images in my project_dir/public directory.

**Comment by Aderemi-Adesada** [comment]
I am experiencing the same problem. my cli app is fast everywhere else except in Tauri

**Reply by Miloszzz2** [reply]
Hi,
I've fixed my problem with changing python to c#. Now it is working so much faster.

**Reply by Aderemi-Adesada** [reply, marked as answer]
I fixed mine by rewriting in Go-lang, seems like python binaries really has bad load time.
